project/pom.xml

```diff
@@ -206,6 +206,7 @@
     <maven.site.url.base>gitsite:git@github.com/jclouds/jclouds-maven-site.git</maven.site.url.base>
     <clojure.version>1.3.0</clojure.version>
     <guava.version>16.0.1</guava.version>
+    <guava.osgi.import>com.google.common.*;version="[16.0.1,19.0.0]"</guava.osgi.import>
```

Comment:
FTR, these are the Guava versions we are compatible with (failed builds are due to transient issues):
https://jclouds.ci.cloudbees.com/job/jclouds-guava-guice-compat/